modules/BioQC.R

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env R
```

Comment:
If you put this file in the `bin` directory and make it executable (chmod +x) it will automatically be added to the `PATH` variable by nextflow.